### ISSUE TITLE
[FR] Remember Focused folder per Workspace

### ISSUE THREAD

**Issue by pdelre** [ozntel/file-tree-alternative#87]
I do not know the limitations of the native Workspace plugin, but it would be nice to have this plugin work with Workspaces by storing the "Focus on Folder"'s state within a workspace.

User story:
- When I switch to the "Main" workspace, I want the panel to be Focused on the root folder (unfocused).
- When I am in the "Projects" workspace, I want the panel to be Focused on the `Projects/` folder.

**Comment by itaikatz** [comment]
I second this request! I keep a separate workspace for each project and I'd love if this plugin remembered my "Focus on Folder" setting for each workspace.

**Comment by vzeazy** [comment]
Would be amazing.

**Comment by mvriel** [comment]
I agree, this is 'the' missing thing for me at the moment working with workspaces. I have a 'Tabletop' workspace and I would love for my Tabletop folder to be the only one that is expanded when I open that workspace. Now it sticks to the last opened folder.
